app/Http/Controllers/ServiceProviderController.php

```diff
@@ -10,48 +10,52 @@
 use Illuminate\Support\Facades\DB;
 use Illuminate\Support\Facades\Validator;
 use App\Http\Controllers\ImageController;
+use App\Estate;
 
 class ServiceProviderController extends Controller
 {
-    public function showAll() {
+    public function showAll()
```

Comment:
Assign the appropriate middleware to this method

Reply:
@gabrielAndy please revert this merge. This commit deletes some lines of code from an earlier commit. I asked him to close it earlier. I’m unable to revert with my phone. He needs to update his branch before raising this pr